cli/commands/nodes.go

```diff
@@ -12,17 +12,17 @@ import (
 func nodeStatus(node api.Node) string {
 	if node.Connected {
 		return "online"
-	} else {
-		return "offline"
 	}
+
+	return "offline"
 }
 
 func nodeInitial(node api.Node) string {
 	if node.InitialMember {
 		return fmt.Sprintf("%d / %d", node.NodeNumber, node.Grid.InitialSize)
-	} else {
-		return "-"
```

Comment:
If you want to break apart the if-else, then IMO put the special case first:

```go
if !node.InitialMember {
  return "-"
}

return fmt.Sprintf(...)
```

Reply:
Same as the comment above ^